.eslintrc.js

```diff
@@ -6,14 +6,20 @@ module.exports = {
     commonjs: true,
     es2020: true,
   },
-  extends: ["eslint:recommended", "plugin:@typescript-eslint/eslint-recommended", "plugin:@typescript-eslint/recommended"],
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:prettier/recommended",
+    "prettier/@typescript-eslint",
+  ],
   parserOptions: {
-    ecmaVersion: 2015,
+    ecmaVersion: 2020,
     sourceType: "module",
   },
-  plugins: ["@typescript-eslint"],
+  plugins: ["@typescript-eslint", "prettier"],
   rules: {
-    indent: ["error", 2],
+    indent: ["error", 2, { SwitchCase: 1 }],
```

Comment:
So indentation is 1 for switch statements? Do we have any of these so I can see what it looks like?

Reply:
Ah, thanks for pointing this out - it should actually be removed as we want `prettier` plugin to enforce indentation so that we don't get issues where the auto-code formatter wants code indented one way and ESLint wants it another way.